powerfulseal/node/node_inventory.py

```diff
@@ -108,13 +108,17 @@ def sync(self, driver=None):
             # different groups can have the same IPs,
             # so we need to match to the same nodes
             for ip in ips:
+                self.logger.debug("IP processing: %s" %(ip))
```

Comment:
(Very) minor nitpicking:

```py
self.logger.debug("IP processing: %s", ip)
```

this will be marginally faster, if the logging level is above debug - it won't be doing the string interpolation.
Also, is quicker to write.

Reply:
oh, shoot yes.   copy/paste propagation ...